src/features/search/SearchLoading/index.js

```diff
@@ -0,0 +1,13 @@
+import image from "./icons/icon-spinner.svg";
+import { ImageContainer, Loader, LoaderContainer, LoadingText } from "./styled";
+
+export const SearchLoading = ({ title }) => {
+  return (
+    <LoaderContainer>
```

Comment:
Instead of LoaderContainer I think there will be a Container from common so you can remove this component